lib/cubit/task_cubit.dart

```diff
@@ -6,16 +6,18 @@ import 'package:task_list_app/service/task_service.dart';
 part 'task_state.dart';
 
 class TaskCubit extends Cubit<TaskState> {
-  TaskCubit() : super(TaskInitial());
-  TaskService service = TaskService();
+  final TaskService taskService;
 
-  void addTask({required Task taskName}) async {
+  TaskCubit({TaskService? taskService})
+      : taskService = taskService ?? TaskService(),
+        super(TaskInitial());
+
+  Future<void> addTask({required Task task}) async {
     try {
-      await service.addTask(taskName: taskName);
+      await taskService.addTask(task: task);
```

Comment:
Nice. This reads much better when the named parameter is "task" rather than "taskName." Please resolve this comment.